src/environments/environment.ts

```diff
@@ -0,0 +1,5 @@
+import versions from "../_versions";
```

Comment:
you have environment.prod, environment.staging. What is environment.ts? Is it the dev environment? so it should use versions.versionLong. How do we select environment based on target deployment (ie, how will we use the environment.staging for staging and environment.prod for production)?

Reply:
environment.ts is the default environment configuration. It's commonly used for development purposes. Settings in this file will be used when you're running the project locally or when you haven't specified a particular environment for deployment. You can select the environment based on the target deployment by setting the NODE_ENV environment variable to development, staging, or production, and then the application dynamically imports the corresponding environment.ts, environment.staging.ts, or environment.prod.ts configuration file. This is typically done in the build or run scripts of your package.json, or through environment variable settings in your CI/CD pipeline.

Reply:
Thanks for the extensive explanation. This information should probably be in the README.md here, linking to a common frontend documentation in dev-rel-docs as we would want to make practices consistent between frontend components.

Reply:
@rngadam I've added further documentation to the TypeScript-AppVersion/APPVERSION-SETUP.md document, highlighting the structure and utilization of different environment configuration files. The update includes detailed guidelines on the purpose and usage of environment.ts, environment.staging.ts, and environment.prod.ts, 

I will update Nachet's README.md documentation and link the dev-rel-docs document once those PR changes are approved: https://github.com/ai-cfia/dev-rel-docs/pull/104